packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -19,18 +27,52 @@ export const PendingTransactions = ({
   error: boolean
 }) => {
   const {
+    l1: { network: l1Network },
     l2: { network: l2Network }
   } = useNetworksAndSigners()
+  const { provider } = useWallet()
+
+  const bgClassName =
+    l2Network.chainID === ChainId.ArbitrumNova
+      ? 'bg-gray-10'
+      : 'bg-blue-arbitrum'
+
+  const switchNetworkMapping: { [chainId: number]: number } = {
+    [ChainId.ArbitrumOne]: ChainId.ArbitrumNova,
+    [ChainId.ArbitrumNova]: ChainId.ArbitrumOne
+  }
```

Comment:
If we create this little helper (outside of the component for it  not to be re-rendered):

```
function getOtherL2NetworkChainId(chainId: number) {
  if (!isNetwork(chainId).isArbitrumOne && !isNetwork(chainId).isArbitrumNova) {
    throw new Error(`[getOtherL2NetworkChainId] Unexpected chain id: ${chainId}`)
  }

  return isNetwork(chainId).isArbitrumOne
    ? ChainId.ArbitrumNova
    : ChainId.ArbitrumOne
}
```

Then we don't have to do the casting to `Number` in the lines below, so it becomes:

```
getNetworkName(getOtherL2NetworkChainId(l2Network.chainID))
```

Reply:
Done